config/default.yml

```diff
@@ -36,7 +36,7 @@ AllCops:
   # -S/--display-style-guide option.
   DisplayStyleGuide: false
   # Extra details are not displayed in offense messages by default. Change
-  # behaviour by overriding ExtraDetails, or by giving the
```

Comment:
I assume these were intentionally written with a preference for the UK spelling. I'm not sure if the codebase is consistently British, though